tests/jsmarionette/client/sockit-to-me/package.json

```diff
@@ -6,7 +6,7 @@
   "main": "build/Release/sockit",
 
   "devDependencies": {
-    "node-gyp": "~0.10.6"
+    "node-gyp": "3.0.1"
```

Comment:
*Here* I would suggest versioning sockit-to-me to `4.0.0` and specifying the engines to `^4.0.0`.